src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
+// *** Solution 1 ***
+/*
+ wall.style.display = 'flex';
+ wall.style.justifyContent = 'center';
+ wall.style.alignItems = 'center';
+ spider.style.position = 'static';
+ */
+
+// *** Solution 2 ***
+const wallStyle = getComputedStyle(wall);
+const spiderStyle = getComputedStyle(spider);
+const wallHalfHeigt = px2number(wallStyle.getPropertyValue('height')) / 2;
+const wallHalfWidth = px2number(wallStyle.getPropertyValue('width')) / 2;
+const spiderOffsetLeft = px2number(spiderStyle.getPropertyValue('height')) / 2;
+const spiderOffsetTop = px2number(spiderStyle.getPropertyValue('width')) / 2;
+
+spider.style.top = (wallHalfHeigt - spiderOffsetTop) + 'px';
+spider.style.left = (wallHalfWidth - spiderOffsetLeft) + 'px';
+
+function px2number(str) {
```

Comment:
It would be great to provide more descriptive and meaningful names, according to these rules:
![image](https://user-images.githubusercontent.com/96409563/236144122-2ed27602-534e-4c1c-93a1-d797e9d7af40.png)